core/src/main/java/org/arquillian/smart/testing/ObjectMapper.java

```diff
@@ -0,0 +1,133 @@
+package org.arquillian.smart.testing;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ObjectMapper {
+
+    public static <T> T mapToObject(Class<T> aClass, Map<String, Object> map) {
```

Comment:
Do we really need to have it public? 
This is used only by the `Configuration` class, so I would move it to the `configuration` package and make it package protected. If we need it also in some other cases, then we can open it to the world.